amazon-linux/Vagrantfile

```diff
@@ -63,7 +63,7 @@ Vagrant.configure("2") do |config|
     samuraiwtf.vm.provider "virtualbox" do |vb|
     # Display the VirtualBox GUI when booting the machine
       vb.gui = true
-      vb.name = "SamuraiWTF-5.2"
+      vb.name = "SamuraiWTF-6.0"
```

Comment:
Is there a reason this wouldn't be the same version as the `Vagrantfile` at the root?

Reply:
should be 5.3 I think